src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -221,6 +227,8 @@ impl CrashReport {
             Some(crash_log.minimized_stack_function_names)
         };
 
+        let onefuzz_version = env!("ONEFUZZ_VERSION").to_string();
```

Comment:
Maybe `onefuzz_version` should be a parameter, and we should do the `env!` lookup where we call `CrashReport::new()`. This way the ctor stays a pure function, the `version` can be changed in tests, &c.